app/views/layouts/application.html.erb

```diff
@@ -68,6 +68,7 @@
     </div>
 
     <%= external_javascript_include_tag "jquery", "2.1.1" %>
+    <%= content_for :js_widget_patch %>
```

Comment:
This seems like a really hacky way to go about this, but I honestly can't think of a better way to fix this problem.  I think we should try to get rid of these layout include tags in a future PR, so that we don't need to load jQuery and bootstrap for pages that don't use it.